ui/src/app/pages/datasets/policies.agent/add/agent.policy.add.component.html

```diff
@@ -98,19 +98,23 @@ <h4>{{ isEdit ? 'Edit Agent Policy' : 'Create Agent Policy'}}</h4>
                          size="medium">
                 <nb-option *ngFor="let option of availableTaps | keyvalue; index as i"
                            [attr.data-orb-qa-id]="option.key"
-                           [value]="option.key">{{ option.key }} <span class="tap-input-text"> {{' | input type: ' + option.value?.input_type }}</span></nb-option>
+                           [value]="option.key">{{ option.key }} <span
+                  class="tap-input-text"> {{' | input type: ' + option.value?.input_type }}</span></nb-option>
               </nb-select>
               <hr/>
             </form>
             <hr/>
           </div>
           <div *ngIf="!availableTaps">
-            <label class="font-weight-bold">There are no <a class="text-link" routerLink="../../../fleet/agents">agents</a> available.</label>
+            <label class="font-weight-bold">There are no <a class="text-link"
+                                                            routerLink="../../../fleet/agents">agents</a>
+              available.</label>
           </div>
           <div *ngIf="!!inputConfigFG">
+<!--            <div>Tap Configuration Options</div>-->
```

Comment:
@iatdaitan should we put some separators for config section and for filter section of tap/input configuration?
I could use some opinions on how to better display the accordion for advanced options, since filter configuration inputs comes after it and not all should be hidden in advanced.